modules/apps/feature-flag/feature-flag-web/src/FeatureFlags.testcase

```diff
@@ -0,0 +1,79 @@
+@component-name = "feature flags"
+definition {
+
+	property portal.acceptance = "true";
+	property portal.upstream = "true";
+	property testray.main.component.name = "Feature Flags";
+
+	setUp {
+		User.firstLoginPG();
+
+		task ("Given the Feature Flags page") {
+			ApplicationsMenu.gotoPortlet(
+				category = "Configuration",
+				panel = "Control Panel",
+				portlet = "Instance Settings");
+
+			Click(
+				key_itemName = "Feature Flags",
+				locator1 = "ListGroupItem#ITEM_TEXT");
+		}
+	}
+
+	tearDown {
+		var testPortalInstance = PropsUtil.get("test.portal.instance");
+
+		if (${testPortalInstance} == "true") {
+			PortalInstances.tearDownCP();
+		}
+	}
+
+	@description = "LPS-186037 - This tests asserts that a feature flag can be Disabled"
+	@priority = 5
+	test CanBeDisabled {
+		property portal.acceptance = "true";
+
+		task ("And a feature flag is enabled") {
+			Click(
+				key_labelName = "Disabled",
+				locator1 = "CalendarLookAndFeelConfiguration#TOGGLE_SWITCH_LABEL");
+		}
+
+		task ("When the administrator disables a feature flag") {
+			Click(
+				key_labelName = "Enabled",
+				locator1 = "CalendarLookAndFeelConfiguration#TOGGLE_SWITCH_LABEL");
```

Comment:
If this path is the only path that works for our usecase in Feature Flags, then make a copy of this and create a new path file atleast in `modules/apps/feature-flag/feature-flag-test/src/testFunctional/paths/FeatureFlags.path`

Reply:
done